maybe_later.gemspec

```diff
@@ -28,5 +28,5 @@ Gem::Specification.new do |spec|
 
   # Uncomment to register a new dependency of your gem
   spec.add_dependency "railties", ">= 6.0.0"
-  spec.add_dependency "concurrent-ruby", "~> 1.1.9"
+  spec.add_dependency "concurrent-ruby", ">= 1.1.9"
```

Comment:
Was this change (and the `Gemfile.lock`) meant to be included? I think the README change is pretty straightforward, just not sure why the dependency change is in here.

Reply:
The dependency change has nothing to do with this PR, which I gave up on. It only shows up here because I made the PR from the main branch of my fork, which I had to update for Rails 7.2 compatibility.